### PR TITLE
English translations for JINS MEME SDK Cordova Plugin

### PR DESCRIPTION
**Pull request by rgbsuede**
I just added some useful information in english and corrected the formatting in some places.  I hope this is helpful!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,39 +22,38 @@
...
 
-* JSONオブジェクトでエラーcodeとmessageを返す
+* Example · JSONオブジェクトでエラーcodeとmessageを返す
 
-	```
-{
-    code: -100,
-    message: "some message" 
-}
+```
```
Comment (duchuyctlk) [inline README.md]: Yu could use ```json to style JSON block. Example:

```json
{
    code: -100,
    message: "some message" 
}
```

**Comment by rgbsuede** [comment]
Hi, I flipped the order of the English and Japanese text ... use whichever you prefer!

**Comment by rgbsuede** [comment]
Closing because there is a much more important improvement (null pointer exception)

**Comment by rgbsuede** [comment]
See #4 for new PR

**Pull request closed without merging**